src/main/java/org/tuberlin/de/deployment/DeploymentImplementation.java

```diff
@@ -136,8 +136,6 @@ public InputStream getZipSource(String entryClass, Map<String, String> clazzes)
         } catch (URISyntaxException | IOException e) {
             LOG.error("Failed to getJarStream", e);
             e.printStackTrace();
-        } finally {
```

Comment:
Nein, das war Absicht. Es kommt sofort ein Fehler, da der FileInputStream ja noch geöffnet ist. Deshalb ist das Cleanup an dieser Stelle nicht richtig. Es sollte erst ausgelöst werden, nachdem der Content der Zip File in den Response OutputStream geschrieben wurde. Das habe ich noch nicht gemacht, aber klar ist, dass der Cleanup an der Stelle oben nicht funktionieren kann...